examples/ExamplesHelper.py

```diff
@@ -0,0 +1,66 @@
+from typing import Dict
+from aperturedb import Connector
+import argparse
+
+DB_HOST = "localhost"
+DB_PORT = 55557
+DB_USER = "admin"
+DB_PASSWORD = "admin"
+
+
+def in_notebook():
+    try:
+        from IPython import get_ipython
+        if 'IPKernelApp' not in get_ipython().config:  # pragma: no cover
+            return False
+    except ImportError:
+        return False
+    except AttributeError:
+        return False
+    return True
+
+
+class ExamplesHelper:
```

Comment:
sure, it is great to add more functionality (in this case, add the ability to change at run time, which would definately be useful in many cases). Can we just add that functionality to dbinfo.py instead of a new file? We will be reusing dbinfo file all over the place, rather than reusing "ExampleHelper" which is not a good name for something that handles information about the db. 

If you are not happy with the dbinfo name, we can change it for something else, but it would be great to have a unified, explicit way to manage db information that we can reuse for all applications, not only demos. 
I think dbinfo is a good name for it, and it will require the least amount of changes. But happy to discuss if that name is not ok.

Reply:
The reason I thought that it should be named into something different because in  a few examples, we specify the number of images to be ingested as a command line param. So that was something like a general app configuration, rather then information about db.

I will change the name to dbinfo for now, We can revisit this once there are more examples to go around.